src/utils/MetaboliteSpectralMatcher.cpp

```diff
@@ -134,7 +134,7 @@ class TOPPMetaboliteSpectralMatcher :
 
     MzMLFile mz_file;
     mz_file.setLogType(log_type_);
-    std::vector<Int> ms_level(1,2);
+    std::vector<Int> ms_level{1,2};
```

Comment:
is this the same as ms_level = {1,2} ?

Reply:
Good question. I'm always confused. Maybe equals is clearer here.

If that was intended anyway.
If we want to load Ms2 only, then the error message should be changed and a less ambiguous code snippet used/ with comment